config/routes.d/staging.horse.json

```diff
@@ -5,7 +5,8 @@
       "^/docs/private-cloud/": "developer.rackspace.com/user-guide.html",
       "^/docs/api-doc-template/": "developer.rackspace.com/user-guide.html",
       "^/docs/billing-api/": "developer.rackspace.com/user-guide.html",
-      "^/docs/sdp/": "developer.rackspace.com/user-guide.html"
+      "^/docs/sdp/": "developer.rackspace.com/user-guide.html",
+      "^/docs/managed-vmware-services/server-virt-customer-handbook/":"https://github.com/rackerlabs/rpc-vmware/server-virt-handbook/master/"
```

Comment:
I believe you want this line to look something like this:

^/docs/managed-vmware-services/server-virt-customer-handbook/":"developer.rackspace.com/user-guide.html"